client/my-sites/plugins/categories/responsive-toolbar-group/style.scss

```diff
@@ -1,25 +1,26 @@
 .responsive-toolbar-group__dropdown {
 	position: relative;
-	width: 100%;
-
-	.responsive-toolbar-group__full-list {
-		width: 100%;
-		display: grid;
-		grid-auto-columns: max-content;
-    	grid-auto-flow: column;
-	}
 
 	.responsive-toolbar-group__grouped-list {
-		width: 100%;
 		justify-content: space-between;
 		flex-wrap: nowrap;
 		display: flex;
 		opacity: 0;
 
+		border-color: rgba( 0, 0, 0, 0 );
+
 		.responsive-toolbar-group__menu-item.is-selected {
 			background-color: #1e1e1e;
 			color: var( --color-text-inverted );
 		}
+
+		// Stretch selected item border to page border
+		>div:first-of-type {
+			margin-left: -8px;
+		}
+		>div:last-of-type {
+			margin-right: -8px;
+		}
```

Comment:
Since this is always highlighted it actually looks kind of off when anything other than the first option is selected.
```suggestion
```